doc/source/train/examples/lightning/lightning_mnist_example.ipynb

```diff
@@ -259,22 +252,29 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 53,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "lightning_config = build_lightning_config_from_existing_code()"
+    "use_gpu = True"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 54,
+   "execution_count": 6,
    "metadata": {},
    "outputs": [],
    "source": [
-    "scaling_config = ScalingConfig(\n",
-    "    num_workers=4, use_gpu=True, resources_per_worker={\"CPU\": 1, \"GPU\": 1}\n",
-    ")\n",
+    "lightning_config = build_lightning_config_from_existing_code(use_gpu=use_gpu)\n",
+    "\n",
+    "if use_gpu:\n",
+    "    scaling_config = ScalingConfig(\n",
+    "        num_workers=4, use_gpu=True, resources_per_worker={\"CPU\": 1, \"GPU\": 1}\n",
+    "    )\n",
+    "else:\n",
+    "    scaling_config = ScalingConfig(\n",
+    "        num_workers=4, use_gpu=False, resources_per_worker={\"CPU\": 1}\n",
+    "    )\n",
```

Comment:
Change this to something like:
```python
scaling_config = ScalingConfig(num_workers=4, use_gpu=use_gpu)
```

Since the `resources_per_worker` should be automatically set.